.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+          node-version: 20
+      - uses: pnpm/action-setup@v4
+        with:
+          version: latest
+      - run: pnpm install --frozen-lockfile
+      - run: pnpm run build
+      
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm run semantic-release
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)